### PR TITLE
Update ml-api-adapter helm chart to 7.1.1, add `ENDPOINT_HEALTH_URL` variable

### PR DESCRIPTION
**Pull request by lewisdaly**
__DON'T MERGE__ - wait for https://github.com/mojaloop/ml-api-adapter/pull/124

- Add a new ENDPOINT_HEALTH_URL variable to enable the new health checks in the ml-api-adapter
- increment the ml-api-adapter helm charts to 7.0.0
- use the upcoming `mojaloop/ml-api-adapter:7.0.0` docker image

### CODE REVIEW

File: ml-api-adapter/chart-service/configs/default.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
 {
     "PORT": {{ .Values.service.internalPort }},
     "ENDPOINT_SOURCE_URL": "http://{{ $centralServicesHost }}:{{ .Values.config.central_services_port }}/participants/{{"{{"}}fsp{{"}}"}}/endpoints",
```
Comment (mdebarros) [inline ml-api-adapter/chart-service/configs/default.json]: Please revert the missing `{`

File: ml-api-adapter/chart-handler-notification/configs/default.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
 {
     "PORT": {{ .Values.service.internalPort }},
     "ENDPOINT_SOURCE_URL": "http://{{ $centralServicesHost }}:{{ .Values.config.central_services_port }}/participants/{{"{{"}}fsp{{"}}"}}/endpoints",
```
Comment (mdebarros) [inline ml-api-adapter/chart-handler-notification/configs/default.json]: Please revert the missing `{`

File: ml-api-adapter/chart-service/configs/default.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
 {
     "PORT": {{ .Values.service.internalPort }},
```
Comment (mdebarros) [inline ml-api-adapter/chart-service/configs/default.json]: Why was this removed?

File: ml-api-adapter/chart-handler-notification/configs/default.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
 {
     "PORT": {{ .Values.service.internalPort }},
```
Comment (mdebarros) [inline ml-api-adapter/chart-handler-notification/configs/default.json]: Why was this removed?

File: ml-api-adapter/chart-service/configs/default.json (hunk 1 above)
Comment (mdebarros) [reply ml-api-adapter/chart-service/configs/default.json]: Or is this being high-lighted incorrectly?

File: ml-api-adapter/chart-handler-notification/configs/default.json (hunk 2 above)
Comment (mdebarros) [reply ml-api-adapter/chart-handler-notification/configs/default.json]: Or is this being high-lighted incorrectly?

File: ml-api-adapter/chart-handler-notification/configs/default.json (hunk 4 above)
Comment (mdebarros) [reply ml-api-adapter/chart-handler-notification/configs/default.json]: Or is this being high-lighted incorrectly?

File: ml-api-adapter/chart-service/configs/default.json (hunk 3 above)
Comment (mdebarros) [reply ml-api-adapter/chart-service/configs/default.json]: Or is this being high-lighted incorrectly?

File: ml-api-adapter/chart-handler-notification/configs/default.json (hunk 2 above)
Comment (lewisdaly) [reply ml-api-adapter/chart-handler-notification/configs/default.json]: I think it's just Github syntax highlighting as it's not technically valid JSON.

The change is on line 6.

File: ml-api-adapter/chart-service/configs/default.json (hunk 1 above)
Comment (lewisdaly) [reply ml-api-adapter/chart-service/configs/default.json]: Same as above

**Comment by mdebarros** [comment]
Erk, you have conflicts that you need to fix0r please @lewisdaly

**Comment by lewisdaly** [comment]
Closing in favour of: https://github.com/mojaloop/helm/pull/199
